File: tests/fixtures/sample.swift
protocol Drawable {
    func draw()
}

enum Status {
    case active
    case inactive
}

struct Point {
    let x: Int
    let y: Int
}

class Shape: Drawable {
    private let name: String

    init(name: String) {
        self.name = name
    }

    func draw() {
        print("Drawing \(name)")
    }
}

func add(a: Int, b: Int) -> Int {
    return a + b
}

File: tests/fixtures/sample.cs
namespace Geometry
{
    interface IDrawable
    {
        void Draw();
    }

    struct Point
    {
        public int X;
        public int Y;
    }

    enum Status
    {
        Active,
        Inactive
    }

    public class Shape : IDrawable
    {
        private string name;

        public Shape(string name)
        {
            this.name = name;
        }

        public void Draw()
        {
            Console.WriteLine($"Drawing {name}");
        }
    }
}
